cosmo_tester/resources/terraform/openstack-vm.tf.template

```diff
@@ -75,6 +75,36 @@ resource "openstack_compute_secgroup_v2" "security_group" {
     ip_protocol = "tcp"
     cidr = "${openstack_networking_subnet_v2.subnet.cidr}"
   }
```

Comment:
can't think of anything that changed; there was some work on the CLI about splitting "profile name" and "manager ip", so perhaps it was broken before; seems to work fine for me right now without those rules, can you try and see please?